build.fsx

```diff
@@ -164,7 +164,7 @@ Target "NuGet" <| fun () ->
 // Generate the documentation
 
 Target "GenerateDocs" <| fun () ->
-    executeFSIWithArgs "docs/tools" "generate.fsx" ["--define:RELEASE"] []
+    executeFSIWithArgs "docs/output" "generate.fsx" ["--define:RELEASE"] []
```

Comment:
The `generate.fsx` script is intended to be in the `docs/tools/` directory, but I have yet to generate the documentation for this project and commit that script.